routing/routing.md

```diff
@@ -0,0 +1,60 @@
+# Routing
+
+This document depicts important information about configuring routing for an existing WSO2 product deployment in a
+ container platform.
+ 
+The following will be discussed in detail in the document.
+
+* [Exposing your application to outside](#exposing-your-application-to-outside )
+* [Configuring hostname](#configuring-hostname)
+* [DNS mapping](#dns-mapping)
+* [Configuring SSL](#configuring-ssl)
+* [Tips on Session Affinity](#tips-on-session-affinity)
+
+### Exposing your application to outside
+
+We recommend the use of [official WSO2 product Helm Charts](https://hub.helm.sh/charts/wso2) for production grade WSO2 
+product deployments in Kubernetes environments. [Kubernetes Ingress resources](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+are used to expose WSO2 product services to outside of the Kubernetes environment.
+
+Kubernetes Ingress Controller is capable of performing the identification of the backend based on the hostname and path.
+It also operates on the application layer of the network stack (HTTP) and can provide features like cookie-based
+session affinity and SSL terminations. 
+
+>WSO2 recommend [Community NGINX](https://kubernetes.github.io/ingress-nginx/) as the Ingress controller for production 
+>grade product deployments in Kubernetes environments.
+
+### Configuring hostname
+
+For all the relevant WSO2 product profile services which need exposing to outside of the Kubernetes cluster have been
```

Comment:
@ThilinaManamgoda I think we should say that `services must be exposed to outside with appropriate hostnames` rather than than saying we already have. Also, no need to mention about paths since, we only deal with hostnames and rely on the default paths provided by the product servers.

And we could also notify the users that hostnames are configurable as per user preference in Helm charts. We could add this as a side note since, this is Kubernetes/Helm specific and is not general to any container platform. For example,
> In official WSO2 product Helm charts, hostnames can be configured via user input (via values.yaml) as per user preference.

WDYT?